src/main/resources/import.sql

```diff
@@ -127,18 +127,18 @@ INSERT INTO value (value, description, header_id, category_id) VALUES ("origin-w
 INSERT INTO value (value, description, header_id, category_id) VALUES ("same-origin", "A referrer will be sent for same-site origins, but cross-origin requests will contain no referrer information", 2, 1);
 INSERT INTO value (value, description, header_id, category_id) VALUES ("strict-origin", "Only set referrer link for HTTPS connections", 2, 1);
 INSERT INTO value (value, description, header_id, category_id) VALUES ("strict-origin-when-cross-origin", "Send a full URL when performing a same-origin request, only send the origin of the document to a-priori as-much-secure destination (HTTPS->HTTPS), and send no header to a less secure destination (HTTPS->HTTP).", 2, 1);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("unsafe-url", "Always set the referrer url, it's recommended to set this to 'default-src' or 'strict-origin'.", 2, 3);
+INSERT INTO value (value, description, header_id, category_id) VALUES ("unsafe-url", "This will always set the referrer url wheter it's a trusted source or not. It's recommended to set this to 'default-src' or 'strict-origin'.", 2, 3);
 
 # =========== Server =============
-INSERT INTO value (value, description, header_id, category_id) VALUES ("*", "Shows potential vulnerable information of what software is being used on the server", 3, 2);
+INSERT INTO value (value, description, header_id, category_id) VALUES ("*", "Shows potential vulnerable information of what software is being used on the server. It's also possible to set this header with misloading information.", 3, 2);
 
 # =========== X-Frame-Options =============
-INSERT INTO value (value, description, header_id, category_id) VALUES ("DENY", "Completely deny loading in frames.", 4, 1);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("SAMEORIGIN", "Only load frames from the same page origin.", 4, 1);
-INSERT INTO value (value, description, header_id, category_id) VALUES ("ALLOW-FROM", "Only load frames from whitelisted sources.", 4, 1);
+INSERT INTO value (value, description, header_id, category_id) VALUES ("DENY", "This header is set to define from which sources the site is allowed to load sources from, consequently avoid clickjacking attacks.", 4, 1);
```

Comment:
This header is set to define from which sources the site is allowed to load sources from, and consequently avoids clickjacking attacks.